### PR TITLE
Avoid unrecoverable failure if the dll can't be found.

### PR DESCRIPTION
**Pull request by hmaarrfk**

### CODE REVIEW

File: ftd2xx/_ftd2xx.py [hunk 1]
```diff
@@ -42,8 +42,9 @@
             _libraries['ftd2xx.dll'] = WinDLL('ftd2xx.dll')
         except OSError as e:
             if e.winerror == 126:
-                sys.stderr.write('Unable to find D2XX DLL. Please make sure ftd2xx.dll or ftd2xx64.dll is in the path\n')
-                sys.exit(1)
+                error_message = e.args[1] + 'Unable to find D2XX DLL. Please make sure ftd2xx.dll or ftd2xx64.dll is in the path.'
+                e.args = (e.args(0), error_message) + e.args[2:]
```
Comment (snmishra) [inline ftd2xx/_ftd2xx.py]: @hmaarrfk Is `args(0)` a typo. Shouldn't it be `args[0]`?

Comment (snmishra) [reply ftd2xx/_ftd2xx.py]: Never mind, I only saw the first commit. Looks good. Thanks for the contribution.

**Comment by hmaarrfk** [comment]
I think this kills python 2 support.

In conda forge, I only applied the patch to python 3.

**Comment by snmishra** [comment]
Works for me in Python 2.7.16

**Comment by hmaarrfk** [comment]
hmm ok I'll add it to the conda forge release then.

**Comment by snmishra** [comment]
@hmaarrfk I had forgotten to push a couple of commits from the release version. I also bumped the version to 1.1.1. If you have already updated conda-forge, please update it again..

**Pull request merged**